src/Sulu/Bundle/WebsiteBundle/Cache/CacheClearer.php

```diff
@@ -69,17 +69,28 @@ public function __construct(
         $this->eventDispatcher = $eventDispatcher;
     }
 
-    public function clear()
+    public function clear(/*?array $tags = []*/)
     {
+        $tags = \func_num_args() >= 1 ? \func_get_arg(0) : null;
+
         if ($this->cacheManager && $this->cacheManager->supportsInvalidate()) {
+            if (!empty($tags) && $this->cacheManager->supportsInvalidateTag()) {
```

Comment:
i think i would expect that nothing is cleared if i pass an empty array here

Reply:
is fine for me but we would need to check if `$tags = null` because then all should be cleared